sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -61,7 +61,7 @@ struct<>
 -- !query
 SHOW TABLES
 -- !query schema
-struct<database:string,tableName:string,isTemporary:boolean>
+struct<namespace:string,tableName:string,isTemporary:boolean>
```

Comment:
at the beginning of this file, let's add ` --SET spark.sql.legacy.keepCommandOutputSchema=false` to make sure the test result is not affected by the config default value.

Reply:
OK. Let's add `--SET spark.sql.legacy.keepCommandOutputSchema=false` to the header of `show-tables.sql`.